### ISSUE TITLE
[ flipper-pkg ] Can't find commands in version after 0.100.0

### ISSUE THREAD

**Issue by cyrusyin97** [facebook/flipper#2710]
Hi, recently I'm using `flipper-pkg` to pack flipper plugins. However when I'm using the latest version 0.103.0 of `flipper-pkg`, I cannot find executable commands in it. 
<img width="467" alt="Screenshot 2021-08-23 at 5 10 39 PM" src="https://user-images.githubusercontent.com/63128320/130421644-2ea4e8ac-53ed-43b6-9e13-050bfe5e9739.png">

I can only get executable commands when I downgrade the version to 0.100.0.
<img width="668" alt="Screenshot 2021-08-23 at 5 12 44 PM" src="https://user-images.githubusercontent.com/63128320/130421917-854f5308-3019-4871-a32d-15185d3d5f2e.png">

I checked the latest version commits but it seems all the command files are good.
Not sure if it's a bug or am I missing anything?

**Comment by SaqibJDev** [comment]
I faced the same issue. Had to downgrade to 0.100.0

**Comment by KevinHu2014** [comment]
Running in local env seems fine, possible some issue happens while packing ?

![image](https://user-images.githubusercontent.com/7204070/130741707-99b8e017-6b6c-47e8-933e-6e250fa5129f.png)

**Comment by KevinHu2014** [comment]
Running locally seems fine, is it possible something mess up during the packing phase?

![image](https://user-images.githubusercontent.com/7204070/130741959-e757a746-022d-4fec-8a75-b218dc4feaa2.png)

**Comment by niusounds** [comment]
me too

**Comment by mweststrate** [comment]
@passy if I try to run `npx flipper-pkg` bundle I get errors from globby, which seems to bundle ESM now (sigh). Would you mind checking if reverting it in D30539761 fixes the issue?

```
 desktop npx flipper-pkg help
(node:81001) [ERR_REQUIRE_ESM] Error Plugin: flipper-pkg [ERR_REQUIRE_ESM]: Must use import to load ES Module: X/sonar/desktop/pkg/node_modules/globby/index.js
require() of ES modules is not supported.
require() of X/desktop/pkg/node_modules/globby/index.js from X/desktop/node_modules/@oclif/config/lib/plugin.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename index.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from X/desktop/pkg/node_modules/globby/package.json.

module: @oclif/config@1.17.0
task: not loading commands, globby not found
plugin: flipper-pkg
root: /Users/mweststrate/fbsource/xplat/sonar/desktop/pkg
See more details with DEBUG=*
(Use `node --trace-warnings ...` to show where the warning was created)
 ›   Warning: flipper-pkg update available from 0.0.0 to 0.92.0.
Utility for building and publishing Flipper plugins

VERSION
  flipper-pkg/0.0.0 darwin-x64 node-v14.17.1

USAGE
  $ flipper-pkg [COMMAND]

COMMANDS
  help  display help for flipper-pkg
  ```
